src/Model/Sluggable/SluggableMethods.php

```diff
@@ -68,7 +68,7 @@ public function getSlug()
      * @param $values
      * @return mixed|string
      */
-    private function generateSlugValue($values)
+    private function generateSlugValue($values, TransliteratorInterface $transliterator)
```

Comment:
I'm not sure i like the idea of injecting the transliterator during the method call.